src/github.com/getlantern/flashlight/genconfig/generate-fallbacks-json.bash

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+function die() {
+  echo $*
+  exit 1
+}
+
+[ -n "$1" ] || die "Usage: '$0 country-code country-code' ...'    (e.g. '$0 nl jp')"
+[ -n "${production_cloudmaster_IP}" ] || die "You need to set the production_cloudmaster_IP environment variable."
+
+for locale
```

Comment:
This is the same shortcut as described below.  If you run `./generate-fallbacks-json.bash nl jp`, `locale` will take the values `nl` and `jp` in respective iterations of the loop.